apps/aecore/src/aec_peers.erl

```diff
@@ -149,7 +147,7 @@ remove(Peer) ->
 %%------------------------------------------------------------------------------
 %% Get list of all peers. The list may be big. Use with caution.
 %% Consider using get_random instead.
-%%------------------------------------------------------------------------------
+%------------------------------------------------------------------------------
```

Comment:
Why?

Reply:
New keyboard, I sometimes the wrong keys which gives interesting effects in vi/vim...